test/tabTest.js

```diff
@@ -29,9 +29,19 @@ describe('tabs', function () {
 
     it('creates a private new tab when signaled', function *() {
       yield this.app.client
-        .ipcSend(messages.SHORTCUT_NEW_FRAME, 'http://www.brianbondy.com', true)
+        .ipcSend(messages.SHORTCUT_NEW_FRAME, 'http://www.brave.com', { isPrivate: true })
         .waitForExist('.tab[data-frame-key="3"]')
-        .waitForVisible('webview[partition]')
+        .waitForVisible('webview[partition="private-1"]')
+    })
+
+    it('creates a partitioned new tab when signaled', function *() {
```

Comment:
Unfortunately those tests in that category depend on each other, should be refactored a bit later.